sycl/test/check_device_code/kernel_arguments_as.cpp

```diff
@@ -0,0 +1,45 @@
+// RUN: %clangxx --sycl -Xclang -fsycl-is-device -emit-llvm %s -S -o %t.ll
```

Comment:
Please, use standard options: "--sycl -Xclang -fsycl-is-device" -> "-fsycl-device-only"